docs/src/how_it_works.md

```diff
@@ -0,0 +1,77 @@
+# How it Works (By Example)
+
+## Example Problem
+
+Let $f$ be a function of position ($\bar{r}$)
```

Comment:
I would maybe use another letter for the position $\bar r\in\mathbb{R}^3$ because IMO something including an $r$ sounds like a radius (especially in the context of a ball). Maybe something like $\boldsymbol{x}$ or maybe better $\boldsymbol{p}$?

Reply:
This might be an electromagnetics-specific notation thing, where I'm very used to seeing $\vec{r}$ used to represent an abstract vector from the origin to some observation point, where no particular coordinate system is yet prescribed.

Reply:
That said, if it’s not universally-understood notation, I can at least clarify the intent in the documentation.